gui/mozregui/utils.py

```diff
@@ -51,48 +51,31 @@ def browse_dialog(self):
             self.line_edit.setPath(path)
 
 
-class RangeSelection(QWidget):
+class BuildSelection(QWidget):
     """
     Allow to select a date, a build id, a release number or an arbitrary
     changeset.
     """
     def __init__(self, parent=None):
         QWidget.__init__(self, parent)
-        layout = QHBoxLayout(self)
-        self._create_widgets()
-        layout.addWidget(self.stacked)
-        layout.addWidget(self.select_combo)
-        self.setLayout(layout)
-
-    def _create_widgets(self):
-        self.stacked = QStackedWidget()
-        self.datew = QDateEdit()
-        self.datew.setDisplayFormat("yyyy-MM-dd")
-        self.stacked.addWidget(self.datew)
-        self.buildidw = QLineEdit()
-        self.stacked.addWidget(self.buildidw)
-        self.releasew = QComboBox()
-        self.releasew.addItems([str(k) for k in sorted(releases())])
-        self.stacked.addWidget(self.releasew)
-        self.revw = QLineEdit()
-        self.stacked.addWidget(self.revw)
-
-        self.select_combo = QComboBox()
-        self.select_combo.addItems(['date', 'buildid', 'release', 'changeset'])
-        self.select_combo.activated.connect(self.stacked.setCurrentIndex)
+        self.helperUi = Ui_BuildSelectionHelper()
```

Comment:
Could you rename self.helperUi to self.ui please ? We use that convention everywhere in the code.